getit/getit.py

```diff
@@ -255,14 +260,21 @@ def main():
             Fore.GREEN + "? " + Style.RESET_ALL +
             "Filename : " + Fore.CYAN + name + Style.RESET_ALL
         )
-    if path == "C:\\Users\\" + getuser() + "\Downloads\getit\\":
+    if (path == "C:\\Users\\" + getuser() + "\Downloads\getit\\"
+            or path == "/Users/" + getuser() + "/Downloads/getit/"
+            or path == "/home/" + getuser() + "/Downloads/getit/"):
         sys.stderr.write(
             Fore.GREEN + "? " + Style.RESET_ALL +
             "File Path : " + Fore.YELLOW + "(" + path + ") " + Style.RESET_ALL
         )
         path = input()
```

Comment:
@tanmaysachan Hey if we store the user input path in some other variable rather than path variable, It will save rechecking of the platform and we can directly assign path if its not null.

Reply:
what say ?

Reply:
Sure. I'm rather busy right now, will make the change tomorrow.

Reply:
No problem